### DISCUSSION TITLE
"Il ne se passe rien" sur l'île !?

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2579] [Idées acceptées / déjà implémentées]
Hello,

J’ai une suggestion pour améliorer nos voyages sur l’île. 
Il y a beaucoup de rapports où il ne se passe rien, c'est frustrant, surtout quand on galère à avancer à des niveaux élevés.

Je pense qu’il serait bien de limiter ou même de supprimer ces rapports vides. Après tout, sur une île noire, on devrait s’attendre à un peu plus d’action ! Même si ça entraîne une augmentation des malus, ce serait plus intéressant que de ne rien avoir du tout. Ça aiderait aussi les joueurs à avancer plus facilement, surtout quand on a besoin de chaque point de rage et d’énergie.

Merci :)

**Comment by romain22222** [comment]
Draft : procède à rajouter des gobelets partout sur l'ile pour la rendre plus divertissante x)

**Reply by BastLast** [reply]
> Draft : procède à rajouter des gobelets partout sur l'ile pour la rendre plus divertissante x)

Suggestion acceptée

**Comment by Ravenclaw-l** [comment]
![ile](https://github.com/user-attachments/assets/40707951-3424-48b8-86cf-1ace6cf479c1)
Hier, entre l'araignée et le golem. C'est too much!

**Comment by StrangerIM** [comment]
Tout à fait d'accord 👍

**Comment by BastLast** [comment]
je viens de coder ceci, ça sortira en v5

**Reply by Ravenclaw-l** [reply]
Tu gères 🙏
